### ISSUE TITLE
Number as parameters are not passed correctly to fn

### ISSUE THREAD

**Issue by Windmolders** [MeLlamoPablo/clapp#7]
Clapp Version: 1.1.1

When you use a command with a number as parameter. Accessing the property gives you the number lowered with the amount of chars in front of it. ( Took me a while to realise what was happening here). 

Codesnippet from my Discord Bot:
```js
var Clapp = require('../modules/clapp-discord');

module.exports = new Clapp.Command({
  name: "listinfo",
  desc: "lets zupa list some info",
  fn: (argv, context) => {
    if(context.msg.author.id == '111897789654421504') {
      console.log(argv.args.guildId);
      context.zbot.listInfo(argv.args.guildId, context.msg.channel);
      //context.zbot.listInfo(context.msg.toString().replace('!zbot listinfo ', '').trim(), context.msg.channel);
    } else {
      context.zbot.printLongMessage('Only Zupa can use this command.', context.msg.channel);
    }
  },
  args: [
    {
      name: 'guildId',
      desc: 'guildId',
      type: 'string',
      required: true,
      default: ''
    }
  ],
  flags: [ ]
});
```

So if I call the following command:
```
!zbot listinfo 256158903245471754
```
The console.log returns me the following
```
256158903245471740
```

The number is lowered by 15 ( which is the amount of chars of '!zbot listinfo ').

**Comment by MeLlamoPablo** [comment]
This is pretty weird. As you probably know, Discord.js needs IDs to be strings because if a number exceeds `Number.MAX_SAFE_INTEGER`, it loses precision. So you're in the right by defining your argument to be `string`. However, apparently it's getting casted to `number` and back to `string` somewhere in the code. So I'll look into it.

As a temporary workaround, appending a letter seems to work:

```js
app.parseInput("test foo 256158903245471754a");
// > 256158903245471754a
```

So you can append a letter to `parseInput` like this:

```js
app.parseInput(input + "a");
```

And in your code:

```js
argv.args.guildId.replace("a", "");
```

Of course, this is a pretty shitty solution and doesn't apply to all commands, but at least it works for now. I apologize, and I'll keep you updated.

**Comment by MeLlamoPablo** [comment]
Apparently this was being caused by minimist and not Clapp. This has been fixed on v1.2.1, which is already on npm.
